glossary/breach.md

```diff
@@ -6,27 +6,22 @@ category = "arvo"
 +++
 
 Continuity on the [Ames](../ames) network occasionally needs to be broken at
-this early stage in order to correct a networking error or make major changes to
-Arvo. These infrequent events are known as
-**breaches** which either cause an individual ship to forget its network message
-history, called a **personal breach**, or cause the entire network to forget its
-history, called a **network breach**.
+this early stage in order to correct a networking error or to make major changes
+to Arvo. These infrequent events are known as **breaches** which either cause an
```

Comment:
This opening paragraph seems to be talking mainly about network breaches, but the paragraph on that says they won't happen anymore. Maybe shift focus to personal breaches right at the start, making network breaches more of a footnote to this section?

We could consider associating "breach" with personal breaches by default, reserving "network breach" for the historical case.

Reply:
https://github.com/urbit/docs/pull/1074/commits/7cf73850873edbb71ac171d3333aceb05478cb27

Reply:
I'd argue the commit is insufficient. The (local) network error case is fair game, but we've never asked/required people to breach to apply an upgrade, and "occasionally" currently feels more like "with increasing rarity".